src/dipoles.jl

```diff
@@ -50,7 +50,7 @@ function radial_dipole_moment(a::Orbital, b::Orbital)
     n == n′ && return 3/2*n*√(n^2-ℓ^2)
 
     A = powneg1(n′-ℓ)/(4*factorial(2ℓ-1))
-    B = √(factorial_ratio(n+ℓ,n-ℓ-1)*factorial_ratio(n′+ℓ-1,n′-ℓ))
+    B = √(factorial(n+ℓ,n-ℓ-1)*factorial(n′+ℓ-1,n′-ℓ))
```

Comment:
Wouldn't the gamma function from SpecialFunctions be better here, for the high `n`/`ell` cases?

Reply:
I am actually unsure, when talking about ratios of factorials. Should be investigated. This is of course related to #2, however, a numerically stable evaluation of the formula will not help for arbitrarily large values of `n`/`ell`, _anyway_, since the dipole moments (in the length gauge) are analytically divergent. This is physically reasonable, since the matrix element is `~ <n' ell' | r | n ell>`, which of course will increase, the larger the values of the quantum numbers are.